src/pimlify-vue/src/store/mutations.ts

```diff
@@ -1,16 +1,17 @@
 import { MutationTree } from 'vuex';
 import State from './state';
-import { OrderMenu } from './mutation-types';
+import { AutoId} from "../auto-id";
 
 const mutations: MutationTree<State> = {
-  setOrderMenus(state, { items }) {
-    state.orderMenus = items;
-  },
-  setCurrentOrderMenu(state, { item }) {
-    state.currentOrderMenu = item;
-  },
   setRestaurants(state, { items }) {
     state.restaurants = items;
+  },
+  addOrderItem(state, {restaurantId,order}){
+    let restaurant = state.restaurants.find(res => res.id === restaurantId);
+    if(restaurant){
+      order.id = AutoId.newId();
+      restaurant.menu.push(order);
```

Comment:
The order should be pushed on the order array.

Reply:
no... that does not do what you think it does :D... it only adds order to the menu and not to the order list

Reply:
Does this make sense? Only if we have orders for one restaurant at a time, right?

Reply:
so... it only is here to add options in the menu... so that we do not need to create them in the firebase console

Reply:
ah, I can remember^^ for now it's fine.